apisix/plugins/limit-count/limit-count-redis-cluster.lua

```diff
@@ -0,0 +1,139 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+local core = require("apisix.core")
+local resty_lock = require("resty.lock")
+local assert = assert
+local setmetatable = setmetatable
+local tostring = tostring
+local require     = require
+local ipairs      = ipairs
+
+
+local _M = {version = 0.3}
+
+
+local mt = {
+    __index = _M
+}
+
+-- https://github.com/steve0511/resty-redis-cluster
+local function new_redis_cluster(conf)
+    local config = {
+        dict_name = "redis_cluster_slot_locks", --shared dictionary name for locks
+        name = "apisix-rediscluster",           --rediscluster name
+        keepalive_timeout = 60000,              --redis connection pool idle timeout
```

Comment:
can we use the default value here?

Reply:
ping  @liuhengloveyou

Reply:
I deleted the dict_name configuration and tested it with no problem.